app/config/config.yml

```diff
@@ -89,6 +89,7 @@ surfnet_stepup_ra_ra:
 
 mopa_bootstrap:
     form:
+        render_optional_text: false
```

Comment:
IIRC, there currently is no of required and optional fields, so a differentiation is not necessary. `'render_required_asterisk' => false,` can be used in the form when it is necessary.